### ISSUE TITLE
Is it possible to render off-screen into a gloo.FrameBuffer?

### ISSUE THREAD

**Issue by drufat** [glumpy/glumpy#100]
I want to run some rendering code on a headless computer without launching a GUI window. Would it be possible to render directly into a `gloo.FrameBuffer` without an `app.Window` followed by app.run()?

**Comment by rougier** [comment]
Short answer: no

Long answer: it depends on the system you're running glumpy on. On linux, there might be some possibilities, but for windows and OSX I'm not sure.

**Comment by drufat** [comment]
Thanks for the explanation.

**Comment by dikr** [comment]
@rougier 
Could you explain the possibilities on Linux a bit more? Where would one need to dig to see if this can be made to work?

**Comment by rougier** [comment]
You could start with EGL: https://devblogs.nvidia.com/parallelforall/egl-eye-opengl-visualization-without-x-server/

**Comment by bhack** [comment]
@rougier Are there any valid EGL python binding to crate that Nvidia ctx?

**Comment by rougier** [comment]
I think vispy solved the problem but I can't find the related issue

**Comment by bhack** [comment]
If is it this https://github.com/vispy/vispy/pull/1464 it is still not merged.

**Comment by daniilhayrapetyan** [comment]
The issue on vispy is closed.

**Comment by rougier** [comment]
I agree. Do you know what exactly was modified to make it work for vispy?
